keras/utils/conv_utils.py

```diff
@@ -85,7 +85,15 @@ def normalize_tuple(value, n, name):
         error_msg += (f'including element {single_value} of '
                       f'type {type(single_value)}')
         raise ValueError(error_msg)
-    return value_tuple
+
+  unqualified_values = [v for v in value_tuple if cmp and cmp(v)]
+
+  if len(unqualified_values) > 0:
+    error_msg += (f' including {unqualified_values}'
+                  f' that does not satisfy the requirement.')
```

Comment:
When user getting this error message, it is unclear to them what is the actual requirement for a valid param (eg it need to be positive or non-negative). The lambda approach makes it hard for showing the actual requirement.

Reply:
It has been resolved. If it looks good, please resolve the conversation.

Reply:
See the comment above

Reply:
Done